datadog/resource_datadog_service_level_objective.go

```diff
@@ -93,6 +93,24 @@ func resourceDatadogServiceLevelObjective() *schema.Resource {
 					},
 				},
 			},
+			"target_threshold": {
+				Description:      "The objective's target in`[0,100]`.",
```

Comment:
```suggestion
				Description:      "The objective's target in `[0,100]`.",
```
one small typo here

Reply:
Oh, thanks. Just pushed a fix.

Reply:
0 and 100 is not valid target value. 
slo target validation is here: https://github.com/DataDog/dogweb/blob/c5d73e5baa851aeaa77ec6188232e4930a397cc1/dogweb/model/service_level_objective.py#L3474-L3475